content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -29,6 +29,21 @@ Kubernetes, on Linux:
 - [Docker Engine](#docker)
 - [Mirantis Container Runtime](#mcr)
 
+{{< note >}}
+Docker container runtime (Dockershim) has been [deprecated](
+https://kubernetes.io/blog/2020/12/02/dockershim-faq/)
+as of [version 1.20](
+https://kubernetes.io/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation)
+and is planned to be removed in by 1.24, therefore to use Docker you need to
+be running an older version of Kubernetes.
+You can check out this [documentation](
+https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/)
+to understand how this deprecation might affect you. For migrating from
+dockershim you can follow [this](
+https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/)
```

Comment:
```suggestion
dockershim you can follow [the migration guide](/docs/tasks/administer-cluster/migrating-from-dockershim/)
```

Reply:
@PranshuSrivastava check the links from the preview, @tengqm 's suggestion is correct

Reply:
@reylejano Okay